### ISSUE TITLE
Link touch targets could be bigger

### ISSUE THREAD

**Issue by jeremyboggs** [scholarslab/scholarslab.github.io#1140]
The spaces between links on the home page are not very big, which fails accessibility testing. Let's make those bigger.
<img width="1647" alt="Screenshot 2024-11-14 at 2 46 45 PM" src="https://github.com/user-attachments/assets/9d43c741-53c6-41a8-a9b0-653f270ac02f">
